src/hooks/requests/useUsers.ts

```diff
@@ -138,13 +138,33 @@ export const useDownloadUsers = () => {
   }
 }
 
-export const useArchiveUser = ({ userId }: { userId?: string }) => {
-  const { mutate: archiveUser, isLoading } = useMutation({
-    mutationKey: ['users', userId],
-    mutationFn: () => {
-      return apiClient.post(endpoints.ARCHIVE_USER, {
-        institution_user_id: userId,
-      })
+export const useArchiveUser = ({
+  institution_user_id,
+}: {
+  institution_user_id?: string
+}) => {
+  const queryClient = useQueryClient()
+  const { mutateAsync: archiveUser, isLoading } = useMutation({
+    mutationKey: ['users', institution_user_id],
+    mutationFn: async (payload: UserStatusType) =>
```

Comment:
**[blocker]**

There should be no need for payload here. As far as I know, we only need to send institution_user_id to the archive user endpoint and this is already passed as an argument to the hooks, so no need to pass it again